nestjs/src/courses/task-verifications/dto/create-task-verification.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { ApiProperty, ApiResponse } from '@nestjs/swagger';
+import { IsNumber, IsOptional } from 'class-validator';
+
+@ApiResponse({})
+export class CreateTaskVerificationDto {
+  constructor(id?: number) {
+    this.id = id;
+  }
+
+  @ApiProperty({ required: false })
```

Comment:
Nest.js recommends to use `@ApiPropertyOptional` for such cases ^^

Reply:
thanks